src/Application.php

```diff
@@ -1,15 +1,15 @@
 <?php declare(strict_types=1);
 
-namespace BehatCsFixer;
+namespace GherkinCsFixer;
```

Comment:
I missed this during my review of the initial PR, but the convention for the namespace is that it should match the composer package name. This is because once you publish this package to Packagist, you have essentially reserved the name and it prevents other package developers from creating conflicts when everyone follows the same convention.

```suggestion
namespace Medology\GherkinCsFixer;
```

To be clear: You don't have to create a Medology folder in src, just change the PSR-4 declaration for src to be Medology\GherkinCsFixer.

Reply:
done